.github/workflows/BesuContainerVerify.sh

```diff
@@ -57,7 +57,7 @@ else
 fi
 
 # For the latest tag check the version match
-if [[ ${TAG} == "latest" && ${CHECK_LATEST} == "true" ]]
+if [[ ${TAG} =~ "latest" && ${CHECK_LATEST} == "true" ]]
```

Comment:
nit, could tighten up the regex slightly: 
```suggestion
if [[ ${TAG} =~ "^latest" && ${CHECK_LATEST} == "true" ]]
```

Reply:
I tried that did not work. Let me check again whether I missed something

Reply:
Found it. I should not use quotes around the pattern. Will fix it.

Reply:
Updated